src/Http/Middleware/AuthenticateWithHeader.php

```diff
@@ -39,8 +39,8 @@ public function process(Request $request, Handler $handler): Response
 
                 $request = $request->withAttribute('apiKey', $key);
                 $request = $request->withAttribute('bypassThrottling', true);
-            } elseif ($token = AccessToken::find($id)) {
-                $token->touch();
```

Comment:
Could we deprecate `touch`? Seems to me that we'll always have a request instance when we need to update the token.

Reply:
True, we could use `touch` for this.

I wasn't sure about deprecating touch. There could be a situation where you want to update the activity but don't have the request object available.

We could use `touch` will nullable request as official API.

Reply:
Let's do that then, it will also make things simpler since we won't need to deprecate anything